source/extensions/filters/http/cache/config.cc

```diff
@@ -1,32 +1,56 @@
 #include "source/extensions/filters/http/cache/config.h"
 
+#include "http_cache.h"
 #include "source/extensions/filters/http/cache/cache_filter.h"
 
 namespace Envoy {
 namespace Extensions {
 namespace HttpFilters {
 namespace Cache {
 
-Http::FilterFactoryCb CacheFilterFactory::createFilterFactoryFromProtoTyped(
-    const envoy::extensions::filters::http::cache::v3::CacheConfig& config,
-    const std::string& stats_prefix, Server::Configuration::FactoryContext& context) {
+namespace {
+template <typename FactoryCtx>
```

Comment:
This no longer needs to be a template - you can use CommonFactoryContext&, because it's a superclass of both the other types.

Reply:
Why was this marked resolved without doing anything?